src/app/workflow/workflow.component.html

```diff
@@ -248,7 +248,12 @@ <h3>
                     <ng-container *ngTemplateOutlet="hostedComponent"></ng-container>
                   </div>
                   <div *ngSwitchCase="WorkflowModel.ModeEnum.DOCKSTOREYML">
-                    <ng-container *ngTemplateOutlet="newComponent"></ng-container>
+                    <div *ngIf="workflow.descriptorType === WorkflowModel.DescriptorTypeEnum.Service">
```

Comment:
What's so special about services?

Reply:
It is a bit of a different scenario, as a service is a dockstore.yml plus an array of files. The dockstore.yml is both a dockstore.yml and a descriptor (kind of). So I thought for simplicity to just keep it with the new component, since that is what we currently do and it works fine.